### DISCUSSION TITLE
Delete section in header

### DISCUSSION THREAD

**Discussion by alliemnguyen** [alshedivat/al-folio#1589] [Q&A] [answered]
![Screenshot 2023-07-21 235357](https://github.com/alshedivat/al-folio/assets/139316543/3f317c4f-2fbf-4442-92af-bc655a44d5e6)
Hello. I'd like to delete the 'publications' and 'repositories' currently on the header and keep the 'more' dropdown menu with those 2 pages inside. How can I do this? I was fiddling with the header.html file but not sure which lines to omit/delete. Thank you!

P/s: My website is [https://alliemnguyen.github.io/](https://alliemnguyen.github.io/)

**Comment by jieliliu** [comment, marked as answer]
Go to _pages/publications.md, change the value of 'nav' from 'true' to 'false', hope it helps.
